config/content.json

```diff
@@ -24,6 +24,7 @@
             "/docs/metrics/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-metrics/",
             "/docs/cloud-identity/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-identity-2.0/",
             "/docs/cloud-monitoring/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-monitoring/",
+            "/docs/dedicated-load-balancers/v2/dev-guide/": "https://github.com/rackerlabs/docs-dedicated-networking/",
```

Comment:
> The doc is only visible to internal users now.

You mean the document is only _linked_ to internal users now. Anyone with a browser can see it. There's a chance it could be crawled by Google, too. I prefer not to think of "it isn't linked from anywhere" as "private." My philosophy is generally to consider any content that's reachable with an unauthenticated HTTP request as "public."

I do agree we can probably still get away with changing the URL without a redirect in this case, though. I'd still prefer to do it soon - I'd rather not break _internal_ users' links and bookmarks, either!

> I also am open to the suggestion that this is not a good idea, and it should be developer-guide.

Well, mostly for consistency with all of the other guides? It looks odd for just this one to have a different path construction. I totally defer to our actual IA experts on link structure, though :wink: